src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/OfficerService.java

```diff
@@ -22,7 +23,12 @@ public OfficersApi getOfficers(String companyNumber) throws ServiceException {
         try {
             var uri = "/company/" + companyNumber + "/officers";
             return apiClientService.getApiKeyAuthenticatedClient().officers().list(uri).execute().getData();
-        } catch (URIValidationException | ApiErrorResponseException e) {
+        } catch (URIValidationException e) {
+            throw new ServiceException("Error Retrieving Officers", e);
+        } catch ( ApiErrorResponseException e) {
+            if (HttpStatus.NOT_FOUND.value() == e.getStatusCode()){
```

Comment:
not sure we should return 0 for a 404, if the service is down then i think an error would be better

Reply:
found out the api can return 404 for 0 officers. that's ok then.